tss-esapi/src/error/return_code/tpm/format_zero/error.rs

```diff
@@ -50,40 +58,40 @@ impl std::error::Error for TpmFormatZeroErrorResponseCode {}
-            TpmFormatZeroError::CommandSize => write!(f, "command commandSizevalue is inconsistent with contents of the command buffer; either the size is not the same as the octets loaded by the hardware interface layer or the value is not large enough to hold a command header"),
-            TpmFormatZeroError::CommandCode => write!(f, "command code not supported"),
-            TpmFormatZeroError::AuthSize => write!(f, "the value of authorizationSizeis out of range or the number of octets in the Authorization Area is greater than required"),
-            TpmFormatZeroError::AuthContext => write!(f, "use of an authorization session with a context command or another command that cannot have an authorization session"),
-            TpmFormatZeroError::NvRange => write!(f, "NV offset+size is out of range"),
-            TpmFormatZeroError::NvSize => write!(f, "Requested allocation size is larger than allowed"),
-            TpmFormatZeroError::NvLocked => write!(f, "NV access locked"),
-            TpmFormatZeroError::NvAuthorization => write!(f, "NV access authorization fails in command actions (this failure does not affect lockout.action)"),
-            TpmFormatZeroError::NvUninitialized => write!(f, "an NV Index is used before being initialized or the state saved by TPM2_Shutdown(STATE) could not be restored"),
-            TpmFormatZeroError::NvSpace => write!(f, "insufficient space for NV allocation"),
-            TpmFormatZeroError::NvDefined => write!(f, "NV Index or persistent object already defined"),
-            TpmFormatZeroError::BadContext => write!(f, "context in TPM2_ContextLoad() is not valid"),
-            TpmFormatZeroError::CpHash => write!(f, "cpHash value already set or not correct for use"),
-            TpmFormatZeroError::Parent => write!(f, "handle for parent is not a valid parent"),
-            TpmFormatZeroError::NeedsTest => write!(f, "some function needs testing."),
-            TpmFormatZeroError::NoResult => write!(f, "returned when an internal function cannot process a request due to an unspecified problem. This code is usually related to invalid parameters that are not properly filtered by the input unmarshaling code."),
-            TpmFormatZeroError::Sensitive => write!(f, "the sensitive area did not unmarshal correctly after decryption - this code is used in lieu of the other unmarshaling errors so that an attacker cannot determine where the unmarshaling error occurred"),
+            TpmFormatZeroError::Initialize => write!(f, "TPM not initialized by TPM2_Startup or already initialized."),
+            TpmFormatZeroError::Failure => write!(f, "Commands not being accepted because of a TPM failure."),
+            TpmFormatZeroError::Sequence => write!(f, "Improper use of a sequence handle."),
+            TpmFormatZeroError::Private => write!(f, "Not currently used."),
+            TpmFormatZeroError::Hmac => write!(f, "Not currently used."),
```

Comment:
These two can be folded `TpmFormatZeroError::Private | TpmFormatZeroError::Hmac`? Not sure if it improves readability that much though :sweat_smile:

Reply:
Nah and besides I think they are just place holders for something that might or might not come.